content/en/docs/contributing/docs/style-guide.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Documentation Style Guide
+linkTitle: Documentation Style Guide
+weight: 20
+description: >
+    Style Guide for Flux Documentation
+---
+## Documentation formatting standards
+
+This page gives writing style guidelines for the Flux documentation. These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+### Use upper camel case for API objects
+
+Do | Don't
+:--| :-----
+| The GitRepository resource is responsible for ... | The Git Repository resource is responsible for ... |
+
+### Use angle brackets for placeholders
+
+Use angle brackets for placeholders. Tell the reader what a placeholder represents, for example
```

Comment:
Which bit in the example tells the reader what the placeholder represents? If it's the "name" in `<name>`, then this could be phrased as "use a word that describes what the placeholder represents".